ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/tests/OpenWorldSandbox.scala

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2020 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.api.testtool.tests
+
+import com.daml.ledger.api.testtool.infrastructure.Allocation.{
+  Participant,
+  Participants,
+  SingleParty,
+  allocate
+}
+import com.daml.ledger.api.testtool.infrastructure.{LedgerSession, LedgerTestSuite}
+import com.digitalasset.ledger.client.binding
+import com.digitalasset.ledger.test.SemanticTests.{Amount, Iou}
+
+class OpenWorldSandbox(session: LedgerSession) extends LedgerTestSuite(session) {
```

Comment:
Not a Ledger API Test Tool test, add a test to the sandbox integration tests rather.